subworkflows/local/bgc.nf

```diff
@@ -0,0 +1,68 @@
+/*
+    Run AMP screening tools
+*/
+include { UNTAR as UNTAR_CSS                       } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_DETECTION                 } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_MODULES                   } from '../../modules/nf-core/modules/untar/main'
+include { ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES } from '../../modules/nf-core/modules/antismash/antismashlitedownloaddatabases/main'
+include { ANTISMASH_ANTISMASHLITE                  } from '../../modules/nf-core/modules/antismash/antismashlite/main'
+
+workflow BGC {
+
+    take:
+    fna     // tuple val(meta), path(PROKKA.out.fna)
+    gff     // tuple val(meta), path(PROKKA.out.gff)
+
+    main:
+    ch_versions = Channel.empty()
+
+    // Check whether user supplies database and/or antismash directory. If not, obtain them via nf-core/modules/antismashlite/antismashlitedownloaddatabases.
+    // Important for future maintenance: For CI tests, only the "else" option below is used. Both options should be tested locally whenever the antiSMASH module gets updated.
+    if ( params.bgc_antismash_databases && params.bgc_antismash_directory ) {
+
+        // Supply databases
+        ch_antismash_databases = Channel
+            .fromPath( params.bgc_antismash_databases )
+            .first()
+
+        // Supply antiSMASH directory
+        ch_antismash_directory = Channel
+            .fromPath( params.bgc_antismash_directory )
+            .first()
+
+    } else {
+
+        print "Warning: No antiSMASH database and/or directory supplied – they will be downloaded by the pipeline."
+
+        // Supply databases
+        ch_css_for_antismash = "https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/antismash/css.tar.gz"
+        ch_detection_for_antismash = "https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/antismash/detection.tar.gz"
+        ch_modules_for_antismash = "https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/antismash/modules.tar.gz"
+
+        UNTAR_CSS ( [ [], ch_css_for_antismash ] )
+        UNTAR_DETECTION ( [ [], ch_detection_for_antismash ] )
+        UNTAR_MODULES ( [ [], ch_modules_for_antismash ] )
+
+        ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES ( UNTAR_CSS.out.untar.map{ it[1] }, UNTAR_DETECTION.out.untar.map{ it[1] }, UNTAR_MODULES.out.untar.map{ it[1] } )
+        ch_versions = ch_versions.mix(ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES.out.versions)
+        ch_antismash_databases = ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES.out.database
+
+        // Supply antiSMASH directory
+        ch_antismash_directory = ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES.out.antismash_dir
```

Comment:
```suggestion

        ch_antismash_databases = ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES.out.database
        ch_antismash_directory = ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES.out.antismash_dir
```

Reply:
I would keep the comment as it indicates what is going on below. Maybe change into `// Supply antiSMASH installation directory`. Because in line 37 we also mention that the database section starts there.

Reply:
I'll add a summary comment for both dirs.